package.json

```diff
@@ -38,10 +39,14 @@
     "postinstall": "husky install"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx,json,sol}": [
+    "*.{js,jsx,ts,tsx,json}": [
       "prettier --write",
       "eslint --fix",
       "cspell --no-must-find-files"
+    ],
+    "*.sol": [
```

Comment:
Logically, this is the same. What's the difference?

Reply:
> Logically, this is the same. What's the difference?

not running `eslint --fix` with *.sol files.
@pavlovcik